docs/source/developer_guide/guides/5_digital_fingerprinting.md

```diff
@@ -290,48 +537,54 @@ After training the generic model, individual user models can be trained​.  Ind
 
 ### Training Stages
 
-#### DFPTraining
+#### Training Stage (`DFPTraining`)
 The `DFPTraining` [examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_training.py](/examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_training.py) trains a model for each incoming `DataFrame` and emits an instance of `morpheus.messages.multi_ae_message.MultiAEMessage` containing the trained model.
```

Comment:
Add paranthesis to link to be consistent with other links or make class name a link.